src/test/java/e2e/pages/ContactsPage.java

```diff
@@ -7,19 +7,23 @@
 import java.util.NoSuchElementException;
 
 public class ContactsPage extends BasePage {
+
     public ContactsPage(WebDriver driver) {
+
         super(driver);
     }
 
+
     @FindBy(xpath ="//div[@class='collapse navbar-collapse']")
     WebElement header;
 
     public boolean confirmLogin(){
         try{
+            //header.isDisplayed();
```

Comment:
need to rollback this change
Now code work isn't correct , because work only true
You can check this use debugger